<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OrionPro Verilog Emulator/Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::NoFocus</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDebugger">
    <property name="title">
     <string>Debugger</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionx1"/>
    <addaction name="actionx2"/>
    <addaction name="actionx3"/>
    <addaction name="actionx4"/>
   </widget>
   <addaction name="menuDebugger"/>
   <addaction name="menuSettings"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionx1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;x1</string>
   </property>
  </action>
  <action name="actionx2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>x&amp;2</string>
   </property>
  </action>
  <action name="actionx3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>x&amp;3</string>
   </property>
  </action>
  <action name="actionx4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>x&amp;4</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
